pyproject.toml

```diff
@@ -12,14 +12,14 @@ license-files = { paths = ["LICENSE"] }
 requires-python = ">=3.9"
 authors = [
     { name = "Petr Splichal", email = "psplicha@redhat.com" },
-]
+    ]
```

Comment:
TOML spirits might haunt you @psss :)

Not sure if we can even configure an IDE or formatter to do this without hacks.

Reply:
Oh! Is `toml` so sensitive?

Reply:
I guess it can parse it fine, otherwise it would be caught in the pre-commit check. It's just that when working on pyproject things, I went through the [documentation](https://toml.io/en/v1.0.0) and all kinds of [pyproject.]toml examples, linters, formatters and never saw this formatting. 

Looking at tools - in Pycharm, I can configure indent and configuration indent, in pyproject-fmt, there is also only `--indent` option.

When you say stay consistent, these brackets might be closer to json then python?

Reply:
Did anyone say TOML? :) https://hitchdev.com/strictyaml/why-not/toml/

Reply:
> Looking at tools - in Pycharm, I can configure indent and configuration indent, in pyproject-fmt, there is also only `--indent` option.

Ok, I see, if this should cause too much troubles, we can revert.

> When you say stay consistent, these brackets might be closer to json then python?

Yeah, for me, in general, everything "under" should be indented and I find those skips back as confusing. But not a big deal here (with python code would be much worse). If you think it's a bad idea we can forget about it.

Reply:
> Did anyone say TOML? :) https://hitchdev.com/strictyaml/why-not/toml/

I don't think anybody's suggesting to use it outside pyproject :)

Since you mentioned strictyaml though - it might be a good replacement for ruamel

Reply:
@psss So how would it work with multiple items in the array?
```
authors = [
    { name = "Petr Splichal", email = "psplicha@redhat.com" },
    { name = "Petr Splichal's twin", email = "psplicha2@redhat.com" }
]
```
```
{
{
[
```
?

Reply:
Sorry, don't understand the question. Anyway, let's not dive into this and finish the important parts...